apps/health_alerts/templates/health_alerts/health_alerts_sign_up.html

```diff
@@ -0,0 +1,89 @@
+{% extends "base.html" %}
+{% load form_tags %}
+
+{% block content %}
+<div class="health-alerts-sign-up-modal columns">
+  <div class="column is-2-tablet"></div>
+  <div class="column py-6">
+    <div class="is-flex is-justify-content-space-between is-vcentered">
+      <h5 class="is-size-5 pt-2 pl-4">
+        <strong>Health Alerts Sign Up Form</strong>
+      </h5>
+      <button id="health-alert-signup-close-btn-hip" data-href="/health-alerts/" class="button btn-no-border-hip pt-4">
+        <i class="close-icon-hip"></i>
+      </button>
+    </div>
+    <div class="px-4 columns is-widescreen">
+    {# Sign Up Form Section #}
+    <form class="column is-half-desktop class="column" action="/health-alerts-subscriber-signup" method="post">
```

Comment:
I think there's an extra `class="column` in there ^

Reply:
I think I didn't indent this section properly.